examples/groq/chat.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\AI\Agent\Agent;
+use Symfony\AI\Platform\Bridge\Groq\Llama;
+use Symfony\AI\Platform\Bridge\Groq\PlatformFactory;
+use Symfony\AI\Platform\Message\Message;
+use Symfony\AI\Platform\Message\MessageBag;
+use Symfony\Component\Dotenv\Dotenv;
+
+require_once dirname(__DIR__, 2).'/vendor/autoload.php';
```

Comment:
The 2 can be removed here:

```suggestion
require_once dirname(__DIR__).'/vendor/autoload.php';
```

Reply:
after this change the example works for me :+1:

Reply:
Ah, thanks. Somehow I got in a bit of a mess with doing a composer install in the monorepo!